index.js

```diff
@@ -0,0 +1,532 @@
+    altRight = new btnRowFive ('AltRight'),
+    arrowLeft = new btnRowFive ('◀'),
+    arrowDown = new btnRowFive ('▼'),
+    arrowRight = new btnRowFive ('▶');
+    controlRight = new btnRowFive ('ControlRight');
+
+
+let pressShift = false;
+let pressCapsLock = false;
+let alterText = false;  
+let switchLanguage = false;
+
+root.prepend(textarea, keyboard);
+body.prepend(root);
+
+// reset event default fisical keybord
+document.onkeydown = (e) => e.code ? false : true;
+
+// switching style key
+const styleKeyDown = (e) => {
+    if(e.code == 'ShiftLeft' || e.code == 'ShiftRight' || e.code == 'CapsLock') return;
+    document.querySelector(`.keyboard .${e.code}`).classList.add('active');
+}
+
+// switching style key
+const styleKeyUp = (e) => {
+    if(e.code == 'ShiftLeft' || e.code == 'ShiftRight' || e.code == 'CapsLock') return;
+    document.querySelectorAll(`.active`).forEach(el => el.classList.remove('active'));
+}
+
+// switching style mouse
+const styleMouseDown = (e) => {
+    if(e.target.classList.contains('ShiftLeft') || e.target.classList.contains('ShiftRight')) return;
+    if(!e.target.classList.contains('btn') && !e.target.classList.contains('special--btn')) return;
+    e.target.classList.add('active');
+}
+
+// switching style mouse
+const styleMouseUp = (e) => {
+    if(e.target.classList.contains('ShiftLeft') || e.target.classList.contains('ShiftRight')) return;
+    document.querySelectorAll(`.active`).forEach(el => el.classList.remove('active'));
+}
+
+// add text in text area for key event;
+const innerTextAreaOnKeyDown = (e) => {
+
+    if(e.code != 'AltLeft') {
+       switchLanguage = false;
+    }
+
+    if(e.code == 'Space') {
+       return textarea.value += '\u0020';
+    }
+
+    if(e.code == 'Tab') {
+        return textarea.value += '\u0009';
+    }
+
+    if(e.code == 'Backspace') {
+        return textarea.value = textarea.value.slice(0, textarea.value.length - 1);
+    }
+
+    if(e.code == 'Enter') {
+        return textarea.value += '\r\n';
+    }
+
+    if (e.code == 'ControlRight' || e.code == 'ControlLeft' ||
+        e.code == 'Win' || e.code == 'AltRight' || e.code == 'ShiftLeft' || e.code == 'ShiftRight' ||
+        e.code == 'AltLeft' || e.code == '◀' || e.code == 'CapsLock' ||
+        e.code == '▼' || e.code == '▶' || e.code == '▲') {
+        return;
+    }
+
+    if(pressShift) {
+        textarea.value += e.key.toUpperCase();
+        document.querySelectorAll('.btn').forEach(el => {
+            if(el.getAttribute('data-rucharcode')) {
+                el.innerHTML = el.innerHTML.toLocaleLowerCase();
+            }
+        });
+        return pressShift = false;
+    }   
+    console.log(e.key)
+    return textarea.value += e.key;
+} 
+
+// add text in text area for mouse event;
+const innerTextAreaOnMouseDown = (e) => {
+    
+    if(!e.target.classList.contains('btn') && !e.target.classList.contains('special--btn')) return;
+    
+    if(e.target.classList.contains('Space')) {
```

Comment:
the better way to delete a lot of if and start using object literals

https://ultimatecourses.com/blog/deprecating-the-switch-statement-for-object-literals